locales/en-human.json

```diff
@@ -290,6 +290,11 @@
   "VERIFY_EMAIL" : "Verify your email address",
   "RESEND_EMAIL_VERIFICATION" : "Resend Verification Email",
   "SWEEP" : "Sweep",
+  "SWEEP_ALL" : "Sweep All",
+  "SWEEPING" : "Sweeping",
+  "SWEEP_WARNING" : "This will sweep all of your active addresses into your default account. This action cannot be undone. Addresses will be archived after being swept.",
```

Comment:
This sounds like I'm sweeping all of my addresses into my default account, instead of just my imported addresses. Also can we say what the default account is? Like: "...into your default account (My Bitcoin Wallet). This action cannot be undone..."